package.json

```diff
@@ -274,6 +274,7 @@
     "eventemitter2": "^5.0.1",
     "farce": "^0.2.6",
     "final-form": "4.11.0",
+    "final-form-arrays": "1.1.2",
```

Comment:
I'm leaving this because we are still using arrays.

I didn't want to hack away all the array logic and make it hard to update to multi-channel support later. This way we can easily update when we get our final-form update in.